lib/unix/bgzf.ml

```diff
@@ -76,10 +77,10 @@ let input_s32 ic =
   let b3 = input_byte ic in
   let b4 = input_byte ic in
   let open Int32 in
-  logor (of_int b1)
-    (logor (shift_left (of_int b2) 8)
-      (logor (shift_left (of_int b3) 16)
-                   (shift_left (of_int b4) 24)))
+  bit_or (of_int_exn b1)
```

Comment:
I'm assuming `bit_or` is the new name for `logor`. @pveber Maybe you could confirm this change.